programs/marginfi/src/instructions/marginfi_account/transfer_account.rs

```diff
@@ -106,3 +110,127 @@ impl<'info> TransferToNewAccount<'info> {
         )
     }
 }
+
+pub fn transfer_to_new_account_pda(
```

Comment:
Same again here, we've mostly been keeping instruction in separate files. Although I know in some files we've stacked them. Don't see any reason to stack there though I guess.

Reply:
If they are very similar we sometimes keep them together e.g. configure_bank has a few basic config ixes and configure_bank_lite also has a few similar ones. Here they are super similar so it seems pretty obvious to keep them in one file.